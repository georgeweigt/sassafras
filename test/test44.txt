* From SAS manual, p. 5634.

data insurance;
input time size type @@;
sizetype=size*type;
datalines;
17 151 0 26 92 0 21 175 0 30 31 0 22 104 0
0 277 0 12 210 0 19 120 0 4 290 0 16 238 0
28 164 1 15 272 1 11 295 1 38 68 1 31 85 1
21 224 1 20 166 1 13 305 1 30 124 1 14 246 1
;
proc reg data=insurance;
model time = size type sizetype;
