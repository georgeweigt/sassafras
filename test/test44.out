                              Analysis of Variance

               DF     Sum of Squares     Mean Square     F Value     Pr > F
     Model      3         1504.41904       501.47301       45.49     0.0000
     Error     16          176.38096        11.02381                       
     Total     19         1680.80000                                       

              Root MSE            3.32021     R-Square     0.8951
              Dependent Mean     19.40000     Adj R-Sq     0.8754
              Coeff Var          17.11450                        

                              Parameter Estimates

                        Estimate     Std Err     t Value     Pr > |t|
          Intercept     33.83837     2.44065       13.86       0.0000
          SIZE          -0.10153     0.01305       -7.78       0.0000
          TYPE           8.13125     3.65405        2.23       0.0408
          SIZETYPE      -0.00042     0.01833       -0.02       0.9821

